internal/rpcapi/terraform1/terraform1.proto

```diff
@@ -882,15 +882,18 @@ message PlannedChange {
 
         repeated AttributePath replace_paths = 11;
 
-        // TODO: Everything else we need for feature-parity with the
-        // existing JSON plan export format.
+        string name = 12;
```

Comment:
```suggestion
        string resource_name = 12;
```

Just to keep the naming scheme in line with the `resource_type` and `resource_mode` above. Optionally, you could even move it up so that it's grouped with the other identifiers but that is totally optional.